EFCore.NamingConventions/NamingConventions/Internal/NamingConventionsOptionsExtension.cs

```diff
@@ -57,13 +64,14 @@ public override string LogFragment
                 => _logFragment ??= Extension._namingConvention switch
                 {
                     NamingConvention.SnakeCase => "using snake-case naming ",
+                    NamingConvention.AllLowerCase => "using lower case naming",
                     _ => ""
                 };
 
             public override long GetServiceProviderHashCode() => Extension._namingConvention.GetHashCode();
 
             public override void PopulateDebugInfo(IDictionary<string, string> debugInfo)
-                => debugInfo["Naming:" + nameof(NamingConventionsExtensions.UseSnakeCaseNamingConvention)]
+                => debugInfo["Naming:" + nameof(NamingConventionsOptionsExtension)]
```

Comment:
This should actually be changed (it wasn't right before either)... It should be:

```suggestion
                => debugInfo["NamingConventions:UseNamingConvention"]
```

At some point we should add a general `UseNamingConvention` builder extension method that accepts that enum, and this would be a `nameof` of that.